### PR TITLE
 🎉 Rename board on the workspace 

### PR DESCRIPTION
**Pull request by mavalroot**
Related to https://tree.taiga.io/project/penpot/task/9101

### CODE REVIEW

File: frontend/src/app/main/ui/workspace/viewport/widgets.cljs [hunk 1]
```diff
@@ -134,15 +132,54 @@
...
+        edition?         (deref edition*)
+
+        local-ref        (mf/use-ref)
+        ref              (d/nilv external-ref local-ref)
+
+        frame-id  (:id frame)
+
+        start-edit
+        (mf/use-fn
+         (mf/deps frame-id)
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/viewport/widgets.cljs]: Missing `edition?` on deps

File: frontend/src/app/main/ui/workspace/viewport/widgets.cljs [hunk 2]
```diff
@@ -152,33 +189,47 @@
...
+          [:div {:class (stl/css :workspace-frame-label)
+                 :style {:color color}
+                 :ref ref
+                 :on-pointer-down on-pointer-down
+                 :on-double-click start-edit
+                 :on-context-menu on-context-menu
+                 :on-pointer-enter on-pointer-enter
+                 :on-pointer-leave on-pointer-leave}
+           (if show-id?
+             (dm/str (dm/str (:id frame)) " - " (:name frame))
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/viewport/widgets.cljs]: `dm/str` looks duplicated

File: frontend/src/app/main/ui/workspace/viewport/widgets.cljs [hunk 3]
```diff
@@ -152,33 +189,47 @@
...
+          [:div {:class (stl/css :workspace-frame-label)
+                 :style {:color color}
+                 :ref ref
+                 :on-pointer-down on-pointer-down
+                 :on-double-click start-edit
+                 :on-context-menu on-context-menu
+                 :on-pointer-enter on-pointer-enter
+                 :on-pointer-leave on-pointer-leave}
+           (if show-id?
+             ((dm/str (:id frame)) " - " (:name frame))
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/viewport/widgets.cljs]: I think this is incorrect, you are creating and string and calling it, which will result to an error. The extra `()` should be removed

Comment (mavalroot) [reply frontend/src/app/main/ui/workspace/viewport/widgets.cljs]: looks like there's a lint error if I remove the extra (), it's old code that was already there so I don't know if the changes are breaking anything

Comment (niwinz) [reply frontend/src/app/main/ui/workspace/viewport/widgets.cljs]: It should be `(dm/str (:id frame) " - " (:name frame))`

**Pull request merged**